go/pgctld/postgresconfig.go

```diff
@@ -199,29 +199,19 @@ func ReadPostgresServerConfig(pgConfig *PostgresServerConfig, waitTime time.Dura
 	var parseErr error
 
 	// Connection settings
-	if pgConfig.Port, parseErr = pgConfig.lookupInt("port"); parseErr != nil {
-		pgConfig.Port = 5432 // default
-	}
+	// Port and unix_socket_directories are NOT read from config file
+	// They are only set via flags/code and passed as PostgreSQL command-line parameters
+	// This ensures backups remain portable across different environments
 	if pgConfig.MaxConnections, parseErr = pgConfig.lookupInt("max_connections"); parseErr != nil {
 		pgConfig.MaxConnections = 100 // default
 	}
 	if val, err := pgConfig.lookupWithDefault("listen_addresses", pgConfig.ListenAddresses); err == nil {
 		pgConfig.ListenAddresses = val
 	}
-	if val, err := pgConfig.lookupWithDefault("unix_socket_directories", pgConfig.UnixSocketDirectories); err == nil {
-		pgConfig.UnixSocketDirectories = val
-	}
 
```

Comment:
This was subtle. We still set this in the config, we just don't _get_ it from the config template. 😅